go_sequencer/test/til/README.md

```diff
@@ -0,0 +1,104 @@
+# Til (Test instructions language)
+Language to define sets of instructions to simulate Hermez transactions (L1 & L2) with real data.
```

Comment:
Can change Hermez transactions to Tokamak, Same for other instances in this file.